symbol/scope.go

```diff
@@ -45,16 +44,16 @@ type Scope struct {
 // Getter return's variable's scope.
 // If a scope doesn't have a variable,
 // Program will search in outer scope.
-func (s *Scope) Get(name string) (Symbol, error) {
+func (s *Scope) Get(name string) Symbol {
 	var scope = s
 	for scope != nil {
 		obj, ok := scope.store[name]
 		if ok {
-			return obj, nil
+			return obj
 		}
 		scope = scope.outer
 	}
-	return nil, errors.New(fmt.Sprintf("%s is not defined", name))
```

Comment:
Question) What is the advantages deleting error?

Reply:
Hmm, why that case is not error..? If user use the symbol which is not defined in any scope, program must occur error. I think this error should be occurred in get function.
So we can show error msg like this symbol is not defined. how about it?
Can we make not defined error in other place?

Reply:
how about following go map convention: return symbol with bool value? like
```
func (s *Scope) Get(name string) (Symbol, bool) {
   ...
}
```
```
// caller
sym, ok := scope.Get("a")
if !ok {
   // throw error or not
}
```

I think it can happen that there are some cases when symbol is not exist in the Scope is not always the error case. And thought that checking whether it is an error or not is depends on the callers context.

![image](https://user-images.githubusercontent.com/18715362/52191000-a9709580-2885-11e9-943c-df4cf6ba70c1.png)

For example, before we trying to update symbol table, we check whether symbol already exist. 
In this case when symbol already exist we should throw error and the case which symbol not exist in the Scope is ok condition. 

So I thought that the case which symbol is not existing in the Scope is always the error case